contracts/Lens/PoolLens.sol

```diff
@@ -302,4 +302,179 @@ contract PoolLens is ExponentialNoError {
                 underlyingPrice: priceOracle.getUnderlyingPrice(address(vToken))
             });
     }
+
+    /**
+     * @dev Struct for Pending Rewards for per market
+     */
+    struct PendingReward {
+        address vTokenAddress;
+        uint256 amount;
+    }
+
+    /**
+     * @dev Struct for Reward of a single reward token.
+     */
+    struct Reward {
+        address distributorAddress;
```

Comment:
Do we need the distributor address?

Reply:
yeah, because on withdrawing rewards we need to call the distributor for the selected reward token